cmd/kritis/kubectl/plugins/resolve/README.md

```diff
@@ -19,6 +19,11 @@ You can then run the plugin:
 kubectl plugin resolve-tags -f <path to file>
 ```
 
+To apply subsitutions using `kubectl apply -f`, you can run:
+```
+kubectl plugin resolve-tags -f <path to file> -a true
```

Comment:
Add an explanation for what  -a stands  before this line or use full name?
As soon as i read it, i thought it stood for --append.

Reply:
For sure, done!